### ISSUE TITLE
Docker / Alpine  - "The type initializer for 'QuestPDF.Drawing.FontManager' threw an exception."

### ISSUE THREAD

**Issue by AlexanderOpran** [QuestPDF/QuestPDF#632]
Hey,

I'm having some issues trying to run QuestPDF inside Docker (using Alpine). I think I have installed all of the dependencies but it's still throwing out the following error: "`The type initializer for 'QuestPDF.Drawing.FontManager' threw an exception.`"

This is the dockerfile that I am using:

```
FROM mcr.microsoft.com/dotnet/aspnet:6.0-alpine

RUN apk add --no-cache icu-libs
ENV DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=false

RUN apk add libgdiplus --no-cache --repository https://dl-3.alpinelinux.org/alpine/edge/community/ --allow-untrusted
RUN apk add libc-dev --no-cache
RUN apk add fontconfig --no-cache

WORKDIR /app
EXPOSE 80
COPY . /app

ENTRYPOINT ["dotnet", "API.dll"]
```

Any help is highly appreciated, thank you! 🤓

**Comment by AlexanderOpran** [comment]
A small update, I have also tried using the following:

`RUN apk add libgdiplus --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/v3.17/community/ --allow-untrusted`

Didn't really change much.

I then did another change; I removed the calls used to register the custom font I was using and then after fixing some other errors that popped out (installed `SkiaSharp.NativeAssets.Linux.NoDependencies` and `HarfBuzzSharp.NativeAssets.Linux.NoDependecies` packages in order to solve the new errors - that were basically complaining about missing these two packages).

Now everything works BUT without any custom fonts. Any ideas on how to get custom font registering to work? Thank you!
